generate_and_send_a_creative_email_using_openai_and_stability_ai/main.bal

```diff
@@ -0,0 +1,80 @@
+import ballerina/io;
+import ballerina/mime;
+import ballerinax/googleapis.gmail;
+import ballerinax/openai.text;
+import ballerinax/stabilityai;
+
+configurable string gmailToken = ?;
+configurable string openAIToken = ?;
+configurable string stabilityAIKey = ?;
+configurable string recipientEmail = ?;
```

Comment:
better to have this as an argument rather than a configurable IMO

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/67/commits/ee50938cc116cc77b3afab1d411343826a263002.